### ISSUE TITLE
SysteminfoBinding throws java.lang.UnsatisfiedLinkError

### ISSUE THREAD

**Issue by kgoderis** [openhab/openhab#2665]
2015-05-29 15:28:20 [ERROR] [b.s.internal.SysteminfoBinding:436 ] - System monitor error: {}org.hyperic.sigar.SigarException: java.lang.UnsatisfiedLinkError: org.hyperic.sigar.Sigar.getNetInterfaceList()[Ljava/lang/String;
at org.hyperic.sigar.SigarProxyCache.invoke(SigarProxyCache.java:220)
at com.sun.proxy.$Proxy49.getNetInterfaceList(Unknown Source)
at org.openhab.binding.systeminfo.internal.SysteminfoBinding.initializeSystemMonitor(SysteminfoBinding.java:419)
at org.openhab.binding.systeminfo.internal.SysteminfoBinding.updated(SysteminfoBinding.java:403)
at org.eclipse.equinox.internal.cm.ManagedServiceTracker$1.run(ManagedServiceTracker.java:276)
at org.eclipse.equinox.internal.cm.SerializedTaskQueue$1.run(SerializedTaskQueue.java:36)

2015-05-29 15:28:20 [INFO ] [.service.AbstractActiveService:169 ] - Systeminfo Refresh Service has been started

**Comment by paulianttila** [comment]
HI Karel,

Did you add Sigar native library to openhab lib folder? See more details from system binding wiki page.

**Comment by kgoderis** [comment]
No I did not but it was part of the oh2 release
I was not aware of the requirement
Suppose I should RTFM :-)

**Comment by teichsta** [comment]
assuming this is no issue anymore?

**Comment by SingleDadLiving** [comment]
This is an issue for me on OpenHAB v2

I followed the wiki exactly. I even tried building my own version. 

My openhab is installed at /opt/openhab on a raspberry Pi 2

I have a lib folder at /opt/openhab/lib
It contains:
* libsigar-arm-linux.so
* log4j.jar
* sigar.jar

and all the files have the proper permissions (pi:pi since that's the user I have running openhab)

I've tested sigar.jar as per the groups post and it works correctly.

I'm currently getting the "Error occurred while reading KPI's
